index.html

```diff
@@ -322,6 +322,36 @@ <h3 class="skills_dropdown_title dropdown_child_class">
                   </li>
                </ul>
            </section>
+
+           <section class="contact" id="contact">
+            <div class="contact__container">
+              <h2 class="contact_header">Contact me</h2>
+              <p class="contact_description">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d love
+                to help with it
+              </p>
+              <form action="https://formspree.io/f/mzbyjjjd" method="post">
+                <ul>
+                  <li>
+                    <label for="name">Name:</label>
+                    <input type="text" id="name" name="name" placeholder="Enter your name here" maxlength="30" required>
+                  </li>
+                  <li>
+                    <label for="email">Email:</label>
+                    <input type="email" id="email" name="email" placeholder="Enter your email here" required>
+                  </li>
+                  <li>
+                    <label for="message">Message:</label>
+                    <textarea id="message" placeholder="Write your message here" maxlength="500" required></textarea>
+                  </li>
+                  <li>
+                    <input type="submit" class="submit_btn" value="Get In Touch" />
+                  </li>
+                </ul>
+              </form>
```

Comment:
- [ ] [OPTIONAL] Great job implementing the recommended changes by the last reviewer, however, kindly ensure that you stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) provided as much as possible. I can see that your chosen template does not have labels but you have included labels on your design, (lines 337, 341, 345) kindly remove them and also fix the width of your email field so your design looks exactly like the one in the template. See below:
### Your design
![your-design](https://user-images.githubusercontent.com/58692590/148848292-98b929a0-5597-40d6-83bd-8a5de126f8e8.png)

### Figma template
![figma](https://user-images.githubusercontent.com/58692590/148848332-ed40c031-c7b0-424a-a708-8333759722b4.png)

Reply:
Thanks a lot for pointing out these issues and making them optional. about the labels, I intentionally added them because of some code reviewers that might probably ask me to add them because they didn't look at the Figma design. I appreciate your effort in thoroughly testing my project. I have fixed the width of my email input on my latest commit.

Reply:
You are welcome @harlexkhal.
Best of luck and happy coding!!!